src/main/scala/beam/agentsim/infrastructure/ParkingManager.scala

```diff
@@ -20,25 +21,25 @@ object ParkingManager {
     destinationUtm: Location,
     activityType: String,
     attributesOfIndividual: AttributesOfIndividual,
-    chargingPreference: ChargingPreference,
+    chargingInquiryData: Option[ChargingInquiryData[String, String]],
```

Comment:
Should `ParkingInquiry` be removed altogether in favor of the one at `beam.agentsim.infrastructure`? That way it avoids confusion later down the road. Or at least some renaming should be done.  In fact it seems that the whole Inquiry/Response setup here can be removed.

Reply:
that's actually what i did - i made `beam.agentsim.infrastructure.ParkingInquiry`. i just forgot to delete this file. ;-)

Reply:
this is logged as a change request on ParkingManager.scala. could you remove it from your review? or am i supposed to do that somehow?

Reply:
that's actually what i did - i made `beam.agentsim.infrastructure.ParkingInquiry`. i just forgot to delete this file. ;-)